### PR TITLE
[HOPSWORKS-320] Disable requests logging for LlapMonitorProxyServlet …

### PR DESCRIPTION
**Pull request by SirOibaf**
…and refactor VariablesFacade

### CODE REVIEW

File: hopsworks-common/src/main/java/io/hops/hopsworks/common/dao/util/VariablesFacade.java [hunk 1]
```diff
@@ -40,37 +40,23 @@ protected EntityManager getEntityManager() {
...
     try {
-      return query.getSingleResult();
-    } catch (NoResultException e) {
-      return null;
-    }
+      Variables var = query.getSingleResult();
+      if (var != null) {
+        return var.getValue();
+      }
+    } catch (NoResultException e) { }
```
Comment (tkakantousis) [inline hopsworks-common/src/main/java/io/hops/hopsworks/common/dao/util/VariablesFacade.java]: @SirOibaf add a logger here

**Comment by SirOibaf** [comment]
Logger added.

**Comment by SirOibaf** [comment]
@tkakantousis I think that INFO is fine, to be honest I think that logging the exception there doesn't make much sense. In my view the `VariablesFacade` should be a dumb class that tries to get a value from the db (if it can find it). If a value cannot be not found, then it's the class using the `VariablesFacade` that should decide if that is acceptable (Like `LlapClusterFacade` does) or it's not, at that point you throw an meaningful exception.

The only reason I see to log the exception there is to help in the debugging. At that point the log level is DEBUG or INFO and you see the exception. But you don't see in a prod environment.

**Comment by SirOibaf** [comment]
Reopen

**Pull request merged**